eacl/target.go

```diff
@@ -77,6 +104,43 @@ func SetTargetECDSAKeys(t *Target, pubs ...*ecdsa.PublicKey) {
 	t.SetBinaryKeys(binKeys)
 }
 
+// SetTargetAccounts sets accounts in Target.
+func SetTargetAccounts(t *Target, accs ...util.Uint160) {
```

Comment:
is this method expected to be used? what cases?

Reply:
We suggest SDK and from my point of view it would be useful for users, for case when you don't need to convert util.Uint160 to user.ID

Reply:
@cthulhu-rider, what do you think about it? (you usually try to drop code whose usage is not clear). i just not sure where it can be used

Reply:
in #605 i deprecate them and provide
```go
func TargetByPublicKeys(ks [][]byte) Target
```
(there are no users introduced yet). With this, i'd provide additional
```
func TargetByUsers(us []user.ID) Target
func TargetByAccounts(hs []util.Uint160) Target
```